app/views/todolists/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Todo Lists</h1>
+<ul>
+  <% @todolists.each do |list| %>
+  <li>
+    <p><%= link_to "#{list.name}", todolist_path(id: list.id) %></p>
```

Comment:
* You don't need to interpolate `"#{list.name}"`, because you may just use `list.name` as the first parameter to `link_to`
* You don't need to pass `id: list.id`, because you may just pass in `list` to `todolist_path`. Rails infers the rest.
* Same comment for all below...